core/templates/pages/splash-page/splash-page.component.spec.ts

```diff
@@ -37,10 +41,18 @@ describe('Splash Page', function() {
   beforeEach(angular.mock.module('oppia', function($provide) {
     $provide.value('WindowRef', windowRefMock);
   }));
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [HttpClientTestingModule]
+    });
+  });
+  beforeEach(angular.mock.module('oppia', function($provide) {
+    $provide.value('UserService', TestBed.get(UserService));
+  }));
   beforeEach(angular.mock.inject(function($injector, $componentController) {
     $timeout = $injector.get('$timeout');
     $q = $injector.get('$q');
-    UserService = $injector.get('UserService');
+    userService = $injector.get('UserService');
```

Comment:
Is this an expected change?

Reply:
Since "UserService" is name of a class being imported , I have changed the variable name to avoid naming clash !